### ISSUE TITLE
Please remove old pypi project

### ISSUE THREAD

**Issue by guettli** [riklaunim/django-ckeditor#60]
Please remove this old pypi project:

https://pypi.python.org/pypi/django-ckeditor-updated

thank you.

**Comment by riklaunim** [comment]
78 downloads in the last day
573 downloads in the last week
2041 downloads in the last month 

it may breake things

**Comment by guettli** [comment]
Yes, you are right. But leaving a more explicit message, that this project gets no updates could be added.

**Comment by riklaunim** [comment]
hm.. is there a way to mark pip package as deprected or something like that? Aside of readme that existing users won't see?
